src/components/Balatro/Balatro.tsx

```
@@ -0,0 +1,237 @@
+    mouseInteraction = true,
+}: BalatroProps) {
+    const containerRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        if (!containerRef.current) return;
+        const container = containerRef.current;
+        const renderer = new Renderer();
+        const gl = renderer.gl;
+        gl.clearColor(0, 0, 0, 1);
+
+        let program: Program;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'program' is only assigned here.


(lint/style/useConst)